config/features.yaml

```diff
@@ -61,3 +61,19 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+ip_address:
+        query_string: 'source_short:"LOG"'
+        attribute: 'ipv4'
+        store_as: 'ip_address'
+        re: '\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}'
```

Comment:
This will catch invalid IPs - 666.789.000.999 for example. Also, if there already is an ipv4 attribute, why do we need to extract it again?